src/Benchmarks/Data/DapperDb.cs

```diff
@@ -40,7 +40,7 @@ public DapperDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<Ap
         async Task<World> ReadSingleRow(DbConnection db)
         {
             return await db.QueryFirstOrDefaultAsync<World>(
-                    "SELECT id, randomnumber FROM world WHERE Id = @Id",
+                    "SELECT id, randomnumber FROM world WHERE id = @Id",
```

Comment:
I think this was required for compat with a particular DB or driver, but either way it seems better all-lowercase.